meetings/2023-09-06.md

```diff
@@ -0,0 +1,164 @@
+* WT: Thanks TBL for the input. Discussion on the GH issue was about MIT license, which is what was incorporated into the charter, but I think it is definitely possible to add public domain for snippets. 
+* TBL: Happy with MIT license. But CC0 for code snippets in specs or tutorials. 
+* SC: I think it's a work item.
+* TBL: No, part of a work item. Code snippets within a document that could have W3C/MIT license, but snippets are CC0. 
+* SC: This licensing issue is about the work item, you are saying distinguishing snippets. That was one of the suggestions in the issue. 
+* RW: I want to reassure TBL that the only issue I see is the notices. Imagine you have software and need notices. We already have a practice of notices, you can take the code somewhere, but then a multitude of notices. IF we accept that notices should be provided, then the CLA would already do what Tim wants. You can co-license with MIT because CLA says "I grant perpetual license with no obligation to reproduce, distribute, etc.." meaning we have it already. This is rather later for WG, when you want to clearly distinguish the specification with the rules where you don't want someone making derivatives of the spec, but you want the code/pseudo code to be reused as much as possible. Similar issue to test-suite, where you don't want to ??? As it stands, the things TBL wants to accomplish can be done now. From a legal standpoint we are clear. Clarify pseudocode is MIT license. 
+* WT: As I understood Tim, he wanted to be able to copy code snippets from specs without the need to also copy a copyright notice into the derivative work. Even snippets in the spec follows MIT, the notice needs to be included. If we want to be able to copy snippet and no notice we need ??? My question to RW is once we go from CG to WG, our specifications become W3C document. W3C license says snippets fall under W3C software license. As we transition, how can this be compatible with snippets being public domain? 
+* RW: We allow specification to have a ??? under W3C license. I'd insist we keep ??? and have a disclaimer. The only thing we need to take is notification, which is a nuisance. It's no issue to transition even if you put code snippets as being public domain. I want to prevent some big corp coming, taking the work, making incompatible changes and claiming they can. What Tim wants is being able to take snippets and put in software with no hassle, but we still should be able to say something is not compatible with what the group consensus was. You have to include an adapted MIT license, I will help you do so. In transitioning to WG, I don't see any issue.
```

Comment:
```suggestion
* RW: We allow specifications to have a ??? under W3C license. I'd insist we keep ??? and have a disclaimer. The only thing we need to take out is notification, which is a nuisance. It's no issue to transition, even if you put code snippets as being public domain. I want to prevent some big corp coming, taking the work, making incompatible changes, and claiming they can. What Tim wants is to be able to take snippets and put them in software with no hassle, but we still should be able to say that something is not compatible with what the group consensus was. You have to include an adapted MIT license; I will help you do so. In transitioning to WG, I don't see any issue.
```